src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -459,6 +459,10 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
     return Array.isArray(value) ? value.join(", ") : value;
   }
 
+  saveAttributeCheckboxState(ev) {
+    localStorage.setItem("devToolsShowAttributes", ev.target.checked);
```

Comment:
This could throw (in safari private mode I think)

Reply:
I think I fixed it.